br/pkg/task/restore.go

```diff
@@ -1240,37 +1240,41 @@ func getStores(ctx context.Context, mgr *conn.Mgr) (stores *http.StoresInfo, err
 	return stores, nil
 }
 
-func EstimateTikvUsage(files []*backuppb.File, maxReplica uint64, storeCnt int) uint64 {
+func EstimateTikvUsage(files []*backuppb.File, replicaCnt uint64, storeCnt uint64) uint64 {
 	if storeCnt == 0 {
 		return 0
 	}
-	var totalSize uint64 = 0
+	if replicaCnt > storeCnt {
+		replicaCnt = storeCnt
```

Comment:
@3pointer just to double check, do we have topology constraint that makes sure no more than 1 replica from the same resign placed on the same store?

Reply:
Yes, our raft implementation make this constraint